### DISCUSSION TITLE
lipsum for Quarto?

### DISCUSSION THREAD

**Discussion by ramiromagno** [quarto-dev/quarto-cli#6231] [Q&A] [answered]
### Description

Is there any thing equivalent of LaTeX's [lipsum](https://github.com/PhelypeOleinik/lipsum) but for quarto? Ideally output-agnostic.

**Comment by dragonstyle** [comment, marked as answer]
I put together a simple version here:

https://github.com/dragonstyle/lipsum

Suggestions or changes very much welcomed!

**Reply by ramiromagno** [reply]
That's awesome!
